suite-native/transactions/src/components/TransactionDetail/NetworkTransactionDetailSummary.tsx

```diff
@@ -46,14 +63,14 @@ export const NetworkTransactionDetailSummary = ({
                     icon={transaction.symbol}
                 />
             )}
-            <TransactionDetailStatusSection txid={txid} accountKey={accountKey} />
             {A.isNotEmpty(transactionOutputAddresses) && (
                 <TransactionDetailAddressesSection
                     addressesType="outputs"
                     addresses={transactionOutputAddresses}
                     onShowMore={onShowMore}
                 />
             )}
+            <VerticalSeparator isMultiAddress={transactionInputAddresses.length > 2} />
```

Comment:
maybe?
```suggestion
            <VerticalSeparator isMultiAddress={transactionOutputAddresses.length > 2} />
```

Reply:
I don't get this, one thing is if we should check length of input or output addresses, other thing is why to check for more than 2 and not just more than 1?

Reply:
Would be awesome to solve it without this param completely, but not sure if there is an easy solution for dynamic height here.

Reply:
For the separator is important how many inputs transaction has, because it is position on the left side of the inputs.

But you are right, the check of `> 1` is better. 59369acb4d

Reply:
just check == 2, please

Reply:
fixed by d57a5ac5f3